app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,7 @@
       <nav>
         <div class="row">
           <div class="col 3">
-            <p class="header-title"><h5>(App Name)</h5></p>
+            <h5 class="header-title">(Twrkr)</h5>
```

Comment:
Let's leave this as App Name for now, Twrkr will be hard to explain to the client